#offset x,y,z boolean
parameters: [-0.6, -0.2, 0.37]
min: [-0.7, -0.3, 0.33]
max: [-0.4, 0.3, 0.65] 
adaptable_parameters: [1, 1, 1]
ConvergenceRate: 3.5
adaptable: 1
nodeNum: 1
